bigchaindb/backend/mongodb/connection.py

```diff
@@ -138,7 +139,7 @@ def _connect(self):
         except (pymongo.errors.ConnectionFailure,
                 pymongo.errors.OperationFailure) as exc:
             logger.info('Exception in _connect(): {}'.format(exc))
-            if "Authentication fail" in str(exc):
```

Comment:
Did this changed or mongodb produces a different message for `SCRAM` and `X509`?
On my machine I do get `OperationFailure: Authentication failed.` when using `SCRAM` authentication